### PR TITLE
Changed pay buckets

### LINKED ISSUE #346: For one-time payment specification in a job post, <50k is not accepted
The form accepts ranges in 50k increments only. It should be possible for a user to either:

1. Have more granularity when specifying one-time payments, or
2. Allow a user to enter any number and don't reset it when they leave the field

### PR DESCRIPTION
**Pull request by iambibhas**
Fixes #346.

 - reduced INR max to 10,00,000, both for one time and recurring
 - changed steps to 5,000 till 1,00,000. 10,000 after that.
 - choosing INR by default

### CODE REVIEW

**Comment by jace** [comment]
INR 1 crore is rather commonplace now, so we should actually be increasing it to 10cr, not decreasing.

**Comment by iambibhas** [comment]
@jace What about first quarter showing till 2 lakh with 10k increment, next 2 quarters till 10 lakh with 50k increment, rest quarter till 10 crore with 1 lakh increment. That'll serve both the one time and recurring payment. Just going by the graph we have.

selection_553

**Comment by iambibhas** [comment]
@jace I updated the buckets to provide more granularity and so that the values can be easily picked. Check them once.

File: hasjob/templates/postjob.html [hunk 1]
```diff
@@ -318,12 +318,14 @@
             $("#recurring-alert").slideUp();
             $("#field-job_pay_currency").show();
             $("#field-cashslider").slideDown();
+            $("#job_pay_currency-0").click();
```
Comment (jace) [inline hasjob/templates/postjob.html]: What does this do?

Comment (iambibhas) [inline hasjob/templates/postjob.html]: Makes INR the default currency.

Comment (jace) [inline hasjob/templates/postjob.html]: That can be fixed in the form server-side. This line is a hack.

Comment (iambibhas) [inline hasjob/templates/postjob.html]: Ah. I see the form. Just removed this and set it on the form.

**Comment by iambibhas** [comment]
With current pay chart buckets, it looks like this - 

selection_613

I'm updating the chart buckets so that they dont overlap - 

selection_614

**Comment by iambibhas** [comment]
I think this PR can be merged now.

File: hasjob/static/js/app.js [hunk 2]
```diff
@@ -558,18 +558,20 @@ window.Hasjob.PaySlider.toNumeric = function(str){
 };
 
 window.Hasjob.PaySlider.range = function(currency){
-  if(currency === '$') {
+  if(currency === 'INR') {
```
Comment (shreyas-satish) [inline hasjob/static/js/app.js]: Space after `if`.

File: hasjob/static/js/app.js [hunk 3]
```diff
@@ -558,18 +558,20 @@ window.Hasjob.PaySlider.toNumeric = function(str){
 };
 
 window.Hasjob.PaySlider.range = function(currency){
-  if(currency === '$') {
+  if (currency === 'INR') {
```
Comment (iambibhas) [inline hasjob/static/js/app.js]: @shreyas-satish done.

**Pull request merged**